profile/src/main/java/rdx/works/profile/data/model/Profile.kt

```diff
@@ -74,6 +87,14 @@ data class Profile(
                 it.common.cryptoParameters == FactorSource.Common.CryptoParameters.babylon
```

Comment:
> This check though was used in main view model to see if main factor source exists, but it currently has no check that main indeed exists, just if it is babylon. Shoudln't this be renamed to `babylonMainDeviceFactorSourceExist` and also check if it is the main one? Could @jakub-rdx you also elaborate too?

@micbakos-rdx Good question. Let's consider scenario upon migration. In such a case we won't have main factor source, while we have only one device factor source (read comments here -> https://github.com/radixdlt/babylon-wallet-ios/blob/main/RadixWallet/Profile/Factor/FactorSourceFlag.swift)
So having just babylon (not main) factor source seems to be legit.

Now question is if we need to change it. We use this check in three places except EnsureBabylonFactorSourceExistUseCase:
1. BackupViewModel 
2. CreateAccountWithLedger
3. WalletViewModel

Then if we change it to check for main, it would mean that existing user after migration won't be able to perform on any of above cases (no main device factor source yet)
Any thoughts @jakub-rdx